packages/editor/src/lib/editor/managers/FocusManager.ts

```diff
@@ -21,6 +21,7 @@ export class FocusManager {
 			(prev, next) => {
 				if (prev.isFocused !== next.isFocused) {
 					next.isFocused ? this.focus() : this.blur()
+					this.updateContainerClass()
```

Comment:
feels like this should be called within `focus`/`blur`, especially because `focusContainer: false` would not focus the container in the Editor API?

Reply:
not sure I follow. this code is intended to keep the container classList in sync with the `instanceState.isFocused` property. It should be triggered whenever that property changes, which is what this guarantees.

Reply:
I think the confusion is sort of related to the `focusContainer` and how the container is where we set the classList.
so, if you manually do _not_ want the default behavior of `focusContainer: true` then, well, i guess the classList shouldn't change, because you're in theory explicitly not focusing the container, right?